app/index.js

```diff
@@ -2,21 +2,21 @@ import React, { View, Text } from 'react-native';
 
 import styles from './styles/default';
 
-class TwoByTwoCards extends React.Component {
+const TwoByTwoCards = React.createClass({
```

Comment:
**<a href='http://eslint.org/docs/rules/react/prefer-es6-class'>react/prefer-es6-class</a>**: Component should use es6 class instead of createClass